docs/StardustDocs/topics/ColumnSelectors.md

```diff
@@ -595,3 +596,27 @@ df.select { (colsOf<Int>() and age).distinct() }
 
 <inline-frame src="resources/org.jetbrains.kotlinx.dataframe.samples.api.Access.columnSelectorsModifySet.html" width="100%"/>
 <!---END-->
+
+### Column Resolvers
+
+`ColumnResolver` is the base type used to access columns within the **Columns Selection DSL**,  
+as well as the return type of columns selection expressions.
```

Comment:
maybe write it like "column(s)", because it's the return type of both the singular and multiple columns dsl

Reply:
I use it as generalized name of Dsl